core/dbt/contracts/graph/parsed.py

```diff
@@ -17,6 +17,7 @@
 
 from dbt.dataclass_schema import dbtClassMixin, ExtensibleDbtClassMixin
 
+from dbt.exceptions import CompilationException
```

Comment:
I noticed no exceptions exist in this file except for the one I introduce. Is that intentional?